src/overlays/actors/ovl_Bg_Mori_Hineri/z_bg_mori_hineri.h

```diff
@@ -6,9 +6,15 @@
 
 struct BgMoriHineri;
 
+typedef void (*BgMoriHineriActionFunc)(struct BgMoriHineri*, GlobalContext*);
+
 typedef struct BgMoriHineri {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x20];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ BgMoriHineriActionFunc actionFunc;
+    /* 0x0168 */ s8 objBankIndex;
```

Comment:
```suggestion
    /* 0x0168 */ s8 moriHineriObjIdx;
```
And corresponding temps should also be updated with a more specific name imo.